candy-machine-core/README.md

```diff
@@ -0,0 +1,5 @@
+# Metaplex Candy Machine Core (a.k.a. Candy Machine V3)
```

Comment:
I'd probably add a little text message explaining that the repo has been moved.

```suggestion
# Metaplex Candy Machine Core (a.k.a. Candy Machine V3)

The Candy Machine Core program and its generated Solita library are now part of a separate `mpl-candy-machine` repository which contains all programs and clients related to Candy Machine V3. Here are some useful links for you to find what you came here for:
```

Reply:
Good point - done!